.claude/skills/mcp-analytics/references/start-here.md

```
@@ -0,0 +1,222 @@
+
+[Privacy & redaction](/docs/mcp-analytics/privacy.md)
+
+---
+
+That's it. You're ready to ship `@posthog/mcp` to production agents – within the beta caveats above.
+
+[Install MCP Analytics](/docs/mcp-analytics/installation.md)
+
+1/7
+
+[**Add @posthog/mcp to your MCP server** ***Required***](#quest-item-add-posthogmcp-to-your-mcp-server)[**See your first events** ***Required***](#quest-item-see-your-first-events)[**Capture what the agent was trying to do** ***Recommended***](#quest-item-capture-what-the-agent-was-trying-to-do)[**Build your first dashboard** ***Recommended***](#quest-item-build-your-first-dashboard)[**Identify the user behind the agent** ***Recommended***](#quest-item-identify-the-user-behind-the-agent)[**Find capability gaps with \`reportMissing\`** ***Recommended***](#quest-item-find-capability-gaps-with-reportmissing)[**Ship safely** ***Required***](#quest-item-ship-safely)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)